swingset/src/main/java/com/nqadmin/swingset/SSBaseComboBox.java

```diff
@@ -1002,7 +1004,7 @@ protected void adjustForNullItem() {
 	 */
 	private void updateRowset() {
 		
-		removeRowSetListener();
+		ssCommon.removeRowSetListener();
```

Comment:
There is also a static method in SSComponentInterface so I could have called SSComponentInterface.removeRowSetListener(this), but the call to SSCommon seemed cleaner in practice.